src/extensions/extensionsApi.ts

```diff
@@ -606,8 +606,10 @@ export async function undeprecateExtensionVersion(extensionRef: string): Promise
  */
 export async function publishExtensionVersion(
   extensionVersionRef: string,
-  packageUri: string,
-  extensionRoot?: string
+  packageUri?: string,
```

Comment:
The param list for this is getting pretty long, and it would be easy to mix up the order of arguments when calling it. Consider  instead:
```
publishExtensionVersion(args: {
  extensionVersionRef: string,
  packageUri?: string,
  extensionRoot?: string,
  repoUri?: string,
  sourceRef?: string
})
```
If you want to get really fancy, you might be able to use a disjoint union type to make it a type error to call this with both packageUri and extensionRoot at the same time